examples/pytorch/graphsage/train_sampling_multi_gpu.py

```diff
@@ -137,7 +131,20 @@ def run(proc_id, n_gpus, args, devices, data):
     th.cuda.set_device(dev_id)
 
     # Unpack data
-    in_feats, n_classes, train_g, val_g, test_g = data
+    n_classes, train_g, val_g, test_g = data
+
+    if args.inductive:
+        train_nfeat = train_g.ndata.pop('features').to(dev_id)
```

Comment:
Will it cause any problem in multi-GPU environment?

Reply:
I tried running the script and it worked fine.